Source/DataSources/CompositePositionProperty.js

```diff
@@ -87,7 +87,7 @@ Object.defineProperties(CompositePositionProperty.prototype, {
  *
  * @param {JulianDate} time The time for which to retrieve the value.
  * @param {Object} [result] The object to store the value into, if omitted, a new instance is created and returned.
```

Comment:
This should also be `Cartesian3`, right?

Reply:
Good catch!